Modules/Relationships/test/Relationships.Domain.Tests/TestHelpers/TestData.cs

```diff
@@ -59,4 +59,11 @@ public static Relationship CreateRelationshipWithRequestedReactivation(IdentityA
         relationship.RequestReactivation(reactivationRequestedBy, DEVICE_1);
         return relationship;
     }
+
+    public static Relationship CreateDecomposedRelationship(IdentityAddress? from = null, IdentityAddress? to = null)
+    {
+        var relationship = CreateTerminatedRelationship(from, to);
+        relationship.Decompose(IDENTITY_1, DEVICE_1);
+        return relationship;
+    }
```

Comment:
Mhhh... If someone passes a custom `from`, then this method will fail, because that from will not be a participant of the relationship.

```suggestion
    public static Relationship CreateDecomposedRelationship(IdentityAddress? from = null, IdentityAddress? to = null, IdentityAddress? decomposedBy = null)
    {
        from ??= IDENTITY_1;
        to ??= IDENTITY_2;
        decomposedBy ??= from;
        
        var relationship = CreateTerminatedRelationship(from, to);
        relationship.Decompose(decomposedBy , DEVICE_1);
        return relationship;
    }
```

Reply:
I had in my mind to add the third parameter because how I implemented always the `IDENTITY_1' is decomposing, but I thought it was sufficient for test purposes. Now if I think about it, for sure it's better this way that the method is more flexible. Changes made.